Debian/8/templates/csv/mount_options.csv

```diff
@@ -1,7 +1,14 @@
 # mountpoint options that should be set
-/home,nosuid,nodev
-/var/log,nosuid,nodev,noexec
-/var/lib,nodev,nosuid
-/var,nosuid,nodev,noexec
-/tmp,nodev,nosuid
+/home,nosuid
+/home,nodev
+/var/log,nosuid
+/var/log,nodev
+/var/log,noexec
+/var/lib,nodev
+/var/lib,nosuid
+/var,nosuid
+/var,nodev
+/var,noexec
+/tmp,nodev
+/tmp,nosuid
```

Comment:
Shouldn't the Debian XCCDF rules also be changed? There could be something referencing the old OVAL IDs.